dockerfiles/cli/scripts/cli.sh

```diff
@@ -91,14 +91,39 @@ cmd_init_reinit_pre_action() {
 }
 
 cmd_start_check_ports() {
+
+  # If dev mode is on, then we also need to check the debug port set by the user for availability
+  if [ "${CHE_DEVELOPMENT_MODE}" = "on" ]; then
+    USER_DEBUG_PORT=$(docker_run --env-file="${REFERENCE_CONTAINER_ENVIRONMENT_FILE}" alpine sh -c 'echo $CODENVY_DEBUG_PORT')
```

Comment:
Just a hint on bash syntax:
If you use `local ` term before declaring variable it will prevent exposing that variable to other functions. It can be much safer way to use local variables that are not intended to be exposed.
Example:
local USER_DEBUG_PORT=....